src/PAModel/Utility/Utility.cs

```diff
@@ -211,7 +211,7 @@ private static bool DontEscapeChar(char ch)
                 (ch == '_') ||
                 (ch == '.') ||
                 (ch == ' ') || // allow spaces, very common.  
-                (ch == '\\'); // Allow directory separators. 
+                (ch == Path.DirectorySeparatorChar); // Allow directory separators. 
```

Comment:
This doesn't feel right... escaping should be agnostic to OS. 
IE, what if we unpack on windows, check into github, and then pack on Linux?

Reply:
Right, yeah. This works for linux->linux round trip, and windows->windows. I think the solution for linux->windows is we need to escape both directory separators, probably.